mmv1/third_party/terraform/services/developerconnect/resource_developer_connect_connection_test.go.tmpl

```diff
@@ -80,4 +85,360 @@ resource "google_developer_connect_connection" "my-connection" {
+
+       
+    service_directory_config {
+      service = "projects/devconnect-terraform-creds/locations/us-central1/namespaces/my-namespace/services/terraform-gle"
+    }
+  }
+
+}
+`, context)
+}
+
+
+func TestAccDeveloperConnectConnection_developerConnectConnectionGitlabEnterpriseUpdate(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix": acctest.RandString(t, 10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDeveloperConnectConnection_GitlabEnterprise(context),
+			},
+			{
+				ResourceName:            "google_developer_connect_connection.my-connection",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"connection_id", "location", "terraform_labels"},
+			},
+			{
+				Config: testAccDeveloperConnectConnection_GitlabEnterpriseUpdate(context),
+			},
+			{
+				ResourceName:            "google_developer_connect_connection.my-connection",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"connection_id", "location", "terraform_labels"},
+			},
+		},
+	})
+}
+
+
+func testAccDeveloperConnectConnection_GitlabEnterprise(context map[string]interface{}) string {
+        return acctest.Nprintf(`
+resource "google_developer_connect_connection" "my-connection" {
+  location = "us-central1"
+  connection_id = "tf-test-tf-test-connection%{random_suffix}"
+
+  gitlab_enterprise_config {
+    host_uri = "https://gle-us-central1.gcb-test.com"
+
+    webhook_secret_secret_version = "projects/devconnect-terraform-creds/secrets/gitlab-enterprise-webhook/versions/latest"
+
+    read_authorizer_credential {
+      user_token_secret_version = "projects/devconnect-terraform-creds/secrets/gitlab-enterprise-read-cred/versions/latest"
+    }
+
+    authorizer_credential {
+      user_token_secret_version = "projects/devconnect-terraform-creds/secrets/gitlab-enterprise-auth-cred/versions/latest"
+    }
+  }
+}
+`, context)
+}
+
+
+func testAccDeveloperConnectConnection_GitlabEnterpriseUpdate(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+resource "google_developer_connect_connection" "my-connection" {
+  location = "us-central1"
+  connection_id = "tf-test-tf-test-connection%{random_suffix}"
+  annotations = {}
+  labels = {}
+  
+  crypto_key_config {
+    key_reference = "projects/devconnect-terraform-creds/locations/us-central1/keyRings/tf-keyring/cryptoKeys/tf-crypto-key"
+  }
+
+  gitlab_enterprise_config {
+    host_uri = "https://gle-us-central1.gcb-test.com"
```

Comment:
Is it possible to update this field with a new value to test if the updating is working or not?